src/main/java/thePackmaster/orbs/PackmasterOrb.java

```diff
@@ -36,10 +38,6 @@ static AbstractOrb getPackLimitedOrb(boolean useCardRng) {
                 orbs.add(new CrescentMoon());
                 orbs.add(new FullMoon());
             }
-            if (pack instanceof StrikesPack) {
```

Comment:
These entries were duplicate with the base game orbs being added above. I don't think having duplicates was intentional, so I removed them.